src/modules/formBuilder/components/FormEditorItemProperties.tsx

```diff
@@ -0,0 +1,182 @@
+        />
+        <TextInput
+          label='Read-only label'
+          value={item.readonlyText || ''}
+          onChange={(e) => {
+            onChangeProperty('readonlyText', e.target.value);
+          }}
+          helperText="Label to display when the item is shown in a read-only form. If not specified, the item's normal label text is shown."
+        />
+        <LabeledCheckbox
+          label='Read-only'
+          checked={item.readOnly}
+          onChange={(e) =>
+            onChangeProperty('readOnly', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <LabeledCheckbox
+          label='Hidden'
+          checked={item.hidden}
+          onChange={(e) =>
+            onChangeProperty('hidden', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <TextInput
+          // TODO(#5776)
+          label='Mapping for custom field key'
+          value={item.mapping?.customFieldKey || ''}
+          onChange={(e) => {
+            onChangeProperty('mapping', { customFieldKey: e.target.value });
+          }}
+        />
+        <LabeledCheckbox
+          label='Warn if empty'
+          checked={item.warnIfEmpty}
+          onChange={(e) =>
+            onChangeProperty(
+              'warnIfEmpty',
+              (e.target as HTMLInputElement).checked
+            )
+          }
+        />
+        <FormSelect
+          label='Data collected about'
+          value={
+            dataCollectedAboutPickList.find(
+              (o) => o.code === item.dataCollectedAbout
+            ) || undefined
+          }
+          options={dataCollectedAboutPickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'dataCollectedAbout',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <FormSelect
+          label='Size'
+          value={
+            inputSizePickList.find((o) => o.code === item.size) || undefined
+          }
+          options={inputSizePickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'size',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <FormSelect
+          label='Disabled Display'
+          value={
+            disabledDisplayPickList.find(
+              (o) => o.code === item.disabledDisplay
+            ) || undefined
+          }
+          options={disabledDisplayPickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'disabledDisplay',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <TextInput
+          label='Pick list reference'
```

Comment:
Only if item type is CHOICE or OPEN_CHOICE

Reply:
Let's discuss with design. We haven't mapped out which properties are type-specific, it's just kind of implicit in the code right now. That would be a good thing to do.

Reply:
In the same vein as https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP?node-id=368-5046#813135090